README.md

```diff
@@ -19,3 +19,34 @@ $ pip install -e .
 ...
 $ SAUCE_USERNAME=<username> SAUCE_ACCESS_KEY=<access-key> copyshop <sauce-labs-test-report-url>
 ```
+
+Feature Support
+---------------
+
+### Desktop Web
+
+- [x] us-west-1 Datacenter
+- [ ] us-east-1 Datacenter
+- [x] eu-central-1 Datacenter
+
+- [x] Capabilities
+- [x] Change URL
+- [x] Find Element By XPath
+- [x] Find Element By name
+- [x] Click on element
+- [x] Clear Element
+- [x] Send Value to Element
+- [x] Implicit Timeouts
+
+### Simulators/Emulators Web
+
+Not tested yet.
+
+### Real Devices
+
+- [x] eu1.appium.testobject.com
```

Comment:
Add us1.appium.testobject.com?

Reply:
I'm not sure yet how to identify which DC has been targeted.
The resulting testing code can still be edited, so I think this is ok to leave that for another PR.

Reply:
We could add a REST call of some kind to figure that out?
This is probably it's own feature.

Reply:
Good to leave as-is for now.

Reply:
Same as for the OS, the DC is part of the report metadata accessible from the API.